consultation_analyser/consultations/views/responses.py

```diff
@@ -23,6 +24,7 @@ def get_filtered_responses(question: models.Question, applied_filters):
     return queryset
 
 
+@waffle_switch("CONSULTATION_PROCESSING", True)
```

Comment:
What does the `True` do? And why do we have it for this one, but none of the other places the decorator is used?

Reply:
Good spot — it's a typo